packages/components/src/theme/ComposerTheme/Components/Radio.ts

```diff
@@ -0,0 +1,84 @@
+import { radioAnatomy as parts } from '@chakra-ui/anatomy';
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme';
+import { PartsStyleInterpolation } from '@chakra-ui/theme-tools';
+
+const sizes: Record<string, PartsStyleInterpolation<typeof parts>> = {
+  sm: ({ theme }) => ({
+    ...theme.textStyles['text-body-meta-regular'],
+  }),
+  md: ({ theme }) => ({
+    ...theme.textStyles['text-body-regular'],
+  }),
+  lg: ({ theme }) => ({
+    ...theme.textStyles['text-body-large-regular'],
+  }),
+};
+
+export const Radio: ComponentMultiStyleConfig = {
+  parts: parts.keys,
+  baseStyle: {
+    control: {
+      color: 'text-light',
+      borderColor: 'ui-element-outline-default',
+      _focus: {
+        boxShadow: 'none',
+        bg: 'ui-element-outline-default',
+        borderColor: 'primary-focus',
+        color: 'text-light',
+        _before: {
+          content: `""`,
+          display: 'inline-block',
+          pos: 'relative',
+          w: '50%',
+          h: '50%',
+          borderRadius: '50%',
+          bg: 'currentColor',
+        },
+        _invalid: {
```

Comment:
Are all of these styles really needed? Our Radio doesn't look like the Chakra-ui...

Check this documentation to see how the focus should behave - https://chakra-ui.com/docs/form/radio